contracts/VotingMachines/GovernanceFormulasInterface.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.4.18;
+
+
+interface GovernanceFormulasInterface {
+
+    function isBoost(bytes32 _proposalId) public view returns (bool);
```

Comment:
isBoost -> isBoosted ?

Reply:
I guess it should be isBoost as it check if a certain proposal should shift to a boost state.

Reply:
Then "shouldBoost" would be better.